### PR TITLE
i18n(es): update `middleware.mdx`

### PR DESCRIPTION
**Pull request by Mardroide**
#### Description 
- Update content

#### Reference
- [a668483](https://github.com/withastro/docs/commit/a668483de3bf957392508d6cc2a86eeb1b1f3fa0)
- [5e4f070](https://github.com/withastro/docs/commit/5e4f070637f51a772a8d029a4967c2c505329ae6)

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Buen trabajo @Mardroide 😁, LGTM ✅

File: src/content/docs/es/guides/middleware.mdx [hunk 1]
```diff
@@ -47,17 +47,21 @@ Esto es un argumento opcional pasado a `onRequest()` que puede contener el objet
 
 ### Almacenando datos en `context.locals`
 
-`context.locals` es un objeto que contiene datos de una `Response` y que se puede manipular dentro del middleware.
+`context.locals` es un objeto que se puede manipular dentro del middleware.
 
 Este objeto `locals` se reenvía a lo largo del proceso de manejo de la solicitud y está disponible como una propiedad de [`APIContext`](/es/reference/api-reference/#contexto-del-endpoint) y [`AstroGlobal`](/es/reference/api-reference/#astrolocals). Esto permite compartir datos entre middlewares, rutas de la API y páginas `.astro`. Esto es útil para almacenar datos específicos de la solicitud, como datos de usuario, durante el paso de renderizado.
 
+:::tip[Propiedades de integración]
```
Comment (Waxer59) [inline src/content/docs/es/guides/middleware.mdx:54]:
```suggestion
:::tip[Propiedades de la integración]
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good work peeps 🚀

**Pull request merged**